WanderLost/WanderLost/Client/Pages/Merchants.razor

```diff
@@ -4,66 +4,66 @@
+  </div>
 }
 
 <div class="row">
-    <div class="col-4">
-        <label for="severRegion">Server Region: </label>
-        <select class="form-select" id="severRegion" @bind="@ServerRegion">
-            <option></option>
-            @foreach (var datacenter in StaticData.ServerRegions)
-            {
-                <option value="@datacenter.Key">@datacenter.Value.Name</option>
-            }
-        </select>
-    </div>
+  <div class="col-4">
+    <label for="severRegion">Server Region: </label>
+    <select class="form-select" id="severRegion" @bind="@ServerRegion">
+      <option></option>
+      @foreach (var datacenter in StaticData.ServerRegions)
+      {
+        <option value="@datacenter.Key">@datacenter.Value.Name</option>
+      }
+    </select>
+  </div>
 
-    <div class="col-4">
-        <label class="control-label" for="server">Server: </label>
-        <select class="form-select" id="server" @bind="@Server">
-            <option></option>
-            @if (!string.IsNullOrWhiteSpace(ServerRegion))
-            {
-                @foreach (var datacenter in StaticData.ServerRegions[ServerRegion].Servers ?? Enumerable.Empty<string>())
-                {
-                    <option>@datacenter</option>
-                }
-            }
-        </select>
+  <div class="col-4">
+    <label class="control-label" for="server">Server: </label>
+    <select class="form-select" id="server" @bind="@Server">
+      <option></option>
+      @if (!string.IsNullOrWhiteSpace(ServerRegion))
+      {
+        @foreach (var datacenter in StaticData.ServerRegions[ServerRegion].Servers ?? Enumerable.Empty<string>())
+        {
+          <option>@datacenter</option>
+        }
+      }
+    </select>
+  </div>
+  @if (!string.IsNullOrWhiteSpace(Server))
+  {
+    <div class="col d-flex justify-content-end mt-auto">
+      <NotificationToggle />
     </div>
-    @if (!string.IsNullOrWhiteSpace(Server))
-    {
-        <div class="col d-flex justify-content-end mt-auto">
-            <NotificationToggle />
-        </div>
-    }
+  }
 </div>
 
-@if(string.IsNullOrWhiteSpace(ServerRegion) || string.IsNullOrWhiteSpace(Server))
+@if (string.IsNullOrWhiteSpace(ServerRegion) || string.IsNullOrWhiteSpace(Server))
 {
-    <h3 class="mt-3">Select a Server</h3>
+  <h3 class="mt-3">Select a Server</h3>
 }
-else if(ActiveData.MerchantGroups.Count > 0)
+else if (ActiveData.MerchantGroups.Count > 0)
 {
-    @foreach(var activeGroup in ActiveData.MerchantGroups.Where(m => m.IsActive))
-    {
-        <Merchant MerchantGroup="@activeGroup" Server="@Server" Votes="@ActiveData.Votes"/>
-    }
-    <FutureMerchantsGrid ActiveData="@ActiveData" Server="@Server" />
+  @foreach (var activeGroup in ActiveData.MerchantGroups.Where(m => m.IsActive))
+  {
+    <Merchant MerchantGroup="@activeGroup" Server="@Server" Votes="@ActiveData.Votes" Region="@activeGroup.MerchantData.Region" />
```

Comment:
why do you pass it again when it's already part of `activeGroup`?

Reply:
Because I tried getting it first from a different var, but then forgot `activeGroup` is a parameter. Thanks for pointing it out, will change